ui/tests/unit/adapters/job-test.js

```diff
@@ -387,6 +389,27 @@ module('Unit | Adapter | Job', function(hooks) {
     assert.notOk(xhr2.aborted, 'Request two was not aborted');
   });
 
+  test('dispatch job encodes payload as base64', async function(assert) {
+    const job = await this.initializeWithJob();
+    job.set('parameterized', true);
+
+    const payload = "I'm a payload 🙂";
```

Comment:
Would it be helpful if we explain why we're specifically making sure to use an emoji here? Maybe editing the `payload` variable name.

Reply:
Text-based tests usually involve adding some non-ASCII characters (like the exec alloc tests). I think it's fairly common in the code base.